### ISSUE TITLE
embedding SBT builds

### ISSUE THREAD

**Issue by cvogt** [cvogt/cbt#64]
It would be great to be able to embed projects built with SBT as subprojects into CBT builds. This will allow to depend on SBT projects by source, (once we have GIT dependencies, which I'll start working on tonight).

This shouldn't be too hard actually. A `cbt.Dependency` subclass that delegates to a SBT build under the hood should do it.

Somebody with solid SBT knowledge wanna pick this up :)? @clhodapp @MasseGuillaume ?

**Comment by MasseGuillaume** [comment]
sbt 1.0 will have a client - server architecture and [sbt-remote-control](https://github.com/sbt/sbt-remote-control) has something similar already.

sbt `dependsOn` work by publishing artifact locally. You could do the same thing with the above api.

**Comment by ScalaWilliam** [comment]
how about an sbt plugin to allow embedding cbt (sub-projects?) within sbt?

**Comment by cvogt** [comment]
@ScalaWilliam yeah thought about it as well. just created a ticket for that: https://github.com/cvogt/cbt/issues/149

**Comment by cvogt** [comment]
I experimented with calling sbt programmatically from cbt here: https://github.com/cvogt/cbt-sbt But it wasn't faster than calling it from the command line if I recall correctly. There also were probably also some other unresolved issues still.
